requirements/typing.txt

```diff
@@ -0,0 +1,17 @@
+mypy==0.982
+torch==1.13.1
```

Comment:
Can you split this PR into 2?

One where you fix the paths and CI config
And another where you upgrade the torch version

Reply:
```suggestion
torch==1.12.0
```